src/Core/InfoBar/IInfoBarManager.cs

```diff
@@ -43,10 +44,21 @@ public interface IInfoBarManager
         /// <returns><see cref="IInfoBar"/></returns>
         IInfoBar AttachInfoBarWithButton(Guid toolWindowGuid, string message, string buttonText, SonarLintImageMoniker imageMoniker);
 
+        /// <summary>
+        /// Attach an info bar with a message and multiple buttons to an existing tool window
+        /// </summary>
+        IInfoBar AttachInfoBarWithButtons(Guid toolWindowGuid, string message, IReadOnlyList<string> buttonTexts, ButtonStyle buttonStyle, SonarLintImageMoniker imageMoniker);
```

Comment:
Is it worth allowing a `ButtonStyle` here? It's just something extra to test, and based on this morning's discussion we'll only use hyperlinks.

Reply:
Regarding different styles for buttons: that would look horrible :D
I'll drop the param and default to hyperlink.